src/bokeh_fastapi/application.py

```diff
@@ -0,0 +1,313 @@
+
+        sign_sessions (bool, optional) :
+            Whether to cryptographically sign session IDs
+
+            Defaults to the current value of the environment variable
+            ``BOKEH_SIGN_SESSIONS``. If ``True``, then ``secret_key`` must
+            also be provided (either via environment setting or passed as
+            a parameter value)
+
+        keep_alive_milliseconds (int, optional) :
+            Number of milliseconds between keep-alive pings
+            (default: {DEFAULT_KEEP_ALIVE_MS})
+
+            Pings normally required to keep the websocket open. Set to 0 to
+            disable pings.
+
+        check_unused_sessions_milliseconds (int, optional) :
+            Number of milliseconds between checking for unused sessions
+            (default: {DEFAULT_CHECK_UNUSED_MS})
+
+        unused_session_lifetime_milliseconds (int, optional) :
+            Number of milliseconds for unused session lifetime
+            (default: {DEFAULT_UNUSED_LIFETIME_MS})
+
+        include_headers (list, optional) :
+            List of request headers to include in session context
+            (by default all headers are included)
+
+        exclude_headers (list, optional) :
+            List of request headers to exclude in session context
+            (by default all headers are included)
+
+        include_cookies (list, optional) :
+            List of cookies to include in session context
+            (by default all cookies are included)
+
+        exclude_cookies (list, optional) :
+            List of cookies to exclude in session context
+            (by default all cookies are included)
+
+        session_token_expiration (int, optional) :
+            Duration in seconds that a new session token is valid
+            for session creation. After the expiry time has elapsed,
+            the token will not be able create a new session
+            (default: {DEFAULT_SESSION_TOKEN_EXPIRATION})
+    """
+
+    def __init__(
+        self,
+        applications: Mapping[str, Application | ModifyDoc] | Application | ModifyDoc,
+        server: FastAPI | None = None,
+        prefix: str | None = None,
+        extra_websocket_origins: Sequence[str] | None = None,
+        secret_key: bytes | None = settings.secret_key_bytes(),
+        sign_sessions: bool = settings.sign_sessions(),
+        generate_session_ids: bool = True,
+        keep_alive_milliseconds: int = DEFAULT_KEEP_ALIVE_MS,
+        check_unused_sessions_milliseconds: int = DEFAULT_CHECK_UNUSED_MS,
+        unused_session_lifetime_milliseconds: int = DEFAULT_UNUSED_LIFETIME_MS,
+        index: str | None = None,
+        xsrf_cookies: bool = False,
+        include_headers: list[str] | None = None,
+        include_cookies: list[str] | None = None,
+        exclude_headers: list[str] | None = None,
+        exclude_cookies: list[str] | None = None,
+        session_token_expiration: int = DEFAULT_SESSION_TOKEN_EXPIRATION,
+        **kwargs: Any
```

Comment:
A bunch of these are currently unused, namely `generate_session_ids`, `index`, `xsrf_cookies`, `session_token_expiration`, and `kwargs`.

Reply:
Yeah, we should in theory add support for all of those. But that doesn't have to be immediately.